src/index.css

```diff
@@ -1,9 +1 @@
 @import url('https://runeapps.org/nis/nis.css');
-
-.app-container {
-  padding: 1rem;
-}
-
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
```

Comment:
Do we not use tailwind anywhere anymore? In that case also remove the tailwind config and the dependency in package.json

Reply:
Ah another thing, I just ran it locally, the resizing is a bit annoying where the highlighted square is moving rather than the abilities sliding to the left.

Reply:
> Ah another thing, I just ran it locally, the resizing is a bit annoying where the highlighted square is moving rather than the abilities sliding to the left.

Attempting to fix this in the following commit. Get rid of your blanks when testing.